test/utils/scaffold.spec.ts

```diff
@@ -47,10 +39,10 @@ describe('Scaffold Utils', () => {
   });
 
   it('should build the scaffold', async () => {
-    const projectDir = getCleanProjectDir('build-scaffold');
+    const projectDir = getCleanProjectDir(pathPrefix, 'build-scaffold');
 
-    await buildScaffold(projectDir, false);
+    await buildScaffold(projectDir, false, false);
 
     expect(fs.readdirSync(projectDir).sort()).toMatchSnapshot();
-  }, 60000);
+  }, 80000);
```

Comment:
Meh. We should probably look into a way soon to not have to increase test timouts regularly...

Reply:
+1